canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -74,27 +78,56 @@ class CheckboxsetControl extends React.Component {
 	}
 
 	render() {
+
+
+		const hidden = this.props.state === STATES.HIDDEN;
+		const disabledTip = this.props.state === STATES.DISABLED;
+
 		let controlValue = this.props.value;
 		if (typeof controlValue === "undefined" || controlValue === null) {
 			controlValue = [];
 		}
 		const checkboxes = [];
 		for (var i = 0; i < this.props.control.values.length; i++) {
+			let tooltip = "";
+			if (this.props.control.description && !this.props.tableControl) {
```

Comment:
Why test for `this.props.control.description` here.  This should be testing for `this.props.control.valueDescs[i]`

Reply:
Done.